### PR TITLE
Manage global login

### PR DESCRIPTION
**Pull request by inpercima**

### CODE REVIEW

File: src/main/resources/public/index.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 <html data-ng-app="app" data-ng-strict-di>
   <head>
+    <base href="/">
```
Comment (inpercima) [inline src/main/resources/public/index.html]: https://docs.angularjs.org/error/$location/nobase

File: src/main/resources/public/partials/navbar.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
         <ul data-uib-dropdown-menu>
           <li><a href="" data-ng-click="vm.open()">Index activities</a></li>
           <li role="separator" class="divider"></li>
-          <li><a href="logout">Logout</a></li>
+          <li><a href="logout" target="_self">Logout</a></li>
```
Comment (inpercima) [inline src/main/resources/public/partials/navbar.html]: https://docs.angularjs.org/guide/$location#html-link-rewriting

**Comment by inpercima** [comment]
@sepe81 You can chck this. After check, I will resolve the conflicts.

**Comment by sepe81** [comment]
I just rebased this on master for better review and testing.

**Pull request merged**